jenkins.sh

```diff
@@ -10,5 +10,7 @@ pip2.7 install sphinx
 make develop extras=[aws,mesos,azure,encryption,cwl]
 export LIBPROCESS_IP=127.0.0.1
 export PYTEST_ADDOPTS="--junitxml=test-report.xml"
+rm -rf /mnt/ephemeral/tmp
```

Comment:
no it doesn't! unless it varies on different platforms. My build just started on a previous slave and the log doesn't show an error.